### DISCUSSION TITLE
[ARCHIVED] Отображение порядка абстракций в IDE

### DISCUSSION THREAD

**Discussion by azinit** [feature-sliced/documentation#36] [Archive]
> Reference: #24 

> @binjospookie
> было бы круто отображать слои прямо на уровне директорий с порядком
>
> т.е. он высоких уровней до низких
>
> не вижу другого способа, кроме добавления каких-то символов в начало директории а-ля 1_ , 2_, 3_, etc :c

**Comment by azinit** [comment]
былоб круто да
Но и вродь с цифрами вариант не оч (хоть он и самый простой)
Но и экстеншен пилить оверхедно как будто

Мб есть в IDEшках (хотя бы в vscode) более простой способ сортировки какой-нить...

**Reply by sergeysova** [reply]
Виктор Пасынок:
> да, цифры выглядят костыльно
> 
> правда, сильно снимают ментальную нагрузку. например:
> * shared
> * entity
> * feature
> 
>если обновился entity, то остаётся поднять что-то в feature (это я опять про пакеты :D )
